### PR TITLE
feat: Added support of kubernetes api server - 6443-tcp

### PR DESCRIPTION
**Pull request by boeboe**
The Kubernetes master server runs a REST API server exposed on 6443 and I would like to use this role to keep my code consistent with other rules already used from here.

### CODE REVIEW

File: rules.tf [hunk 1]
```diff
@@ -213,6 +215,11 @@ variable "auto_groups" {
       ingress_with_self = ["all-all"]
       egress_rules      = ["all-all"]
     }
+    kafka = {
```
Comment (antonbabenko) [inline rules.tf:218]: `kubernetes-api` ?

**Comment by boeboe** [comment]
Hi @antonbabenko

I am using your module to spin up a self hosted modular kubernetes cluster. In order to be able the administer a kubernetes cluster, most of the people use kubectl, a binary executable that under the hood talks JSON REST HTTPS with the apiserver (the configuration server of kubernetes) which is running on one or more master nodes. 

More info at https://kubernetes.io/docs/reference/access-authn-authz/controlling-access/

**Comment by boeboe** [comment]
Hi @antonbabenko 

Any update on this? I would like to add Grafana support as well, but like to wait to see if this one would be accepted first.

File: rules.tf (hunk 1 above)
Comment (antonbabenko) [reply rules.tf:218]: Please replace `kafka` with `kubernetes-api` here.

**Comment by antonbabenko** [comment]
Yes, I commented on it a few days ago. Please fix and I will merge this and another one.

**Comment by boeboe** [comment]
Ok,  I did not see that remark. Typo fixed.

**Comment by antonbabenko** [comment]
Thanks @boeboe !

[v3.5.0](https://github.com/terraform-aws-modules/terraform-aws-security-group/releases/v3.5.0) has been just released with this and your second PR merged.

**Pull request merged**